campaignion_tracking/tests/TrackerTest.php

```diff
@@ -8,29 +11,31 @@
 class TrackerTest extends \DrupalWebTestCase {
 
   /**
-   * Create test nodes.
+   * Create test node with a webform with 1 component.
    */
   public function setUp() {
     parent::setUp();
-    $this->petition = entity_create('node', ['type' => 'petition']);
-    $this->petition->webform = webform_node_defaults();
+    $node['nid'] = 1;
+    $node['type'] = 'petition';
+    $node['title'] = 'Test';
+    $node['language'] = 'und';
+    $node['webform'] = webform_node_defaults();
+    $node['webform']['webform_ajax'] = WEBFORM_AJAX_NO_CONFIRM;
     $component = [
       'type' => 'email',
       'form_key' => 'email',
       'cid' => 1,
       'page_num' => 1,
     ];
     webform_component_defaults($component);
-    $this->petition->webform['webform_ajax'] = WEBFORM_AJAX_NO_CONFIRM;
-    $this->petition->webform['components'][1] = $component;
-    node_save($this->petition);
+    $node['webform']['components'][1] = $component;
+    $this->petition = (object) $node;
   }
 
   /**
-   * Delete test nodes.
+   * Tear down.
    */
   public function tearDown() {
-    node_delete($this->petition->nid);
     parent::tearDown();
   }
```

Comment:
No need to keep this method around just for calling the parent method.

Reply:
Removed